src/SourceBuild/content/eng/tools/init-build.proj

```diff
@@ -119,16 +118,6 @@
     <WriteLinesToFile File="$(CompletedSemaphorePath)BuildXPlatTasks.complete" Overwrite="true" />
   </Target>
 
-  <!-- TODO: Remove this when the .NET 8 artifacts tarball no longer includes MicrosoftAspNetCoreAppRuntimePackageVersion -->
```

Comment:
I don't think this can be removed until we update the previous source build artifacts to be an 8.0 version.  This property is still defined because we are using a 7.0 version of PSB.

Reply:
Yeah you may be right. I think this is the key comment here: https://github.com/dotnet/source-build/issues/3121#issuecomment-1315596188

> [dotnet/installer#14938](https://github.com/dotnet/installer/pull/14938) implements it. It combines my patch with @crummel's PR. For 8.0 we should be able to change back to the original property name (`MicrosoftAspNetCoreAppRuntimePackageVersion`) when the Artifacts tarball no longer includes this name.

I'll check it out.

Reply:
FYI I updated PSB in https://github.com/dotnet/installer/pull/15324 and merge the changes in here to see if it passes CI now.

Reply:
!!! woo!